src/App.module.scss

```diff
@@ -1,25 +1,35 @@
-body {
-    margin: 0;
-    min-height: 100vh;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    background-color: white;
+h1 {
+    font-size: 7vmin;
+    color: var(--heading-color);
+    text-shadow: 1px 1px 2px black;
 }
 
-.banner{
-    width: 100%;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    img {
-        width: 50%;
-        max-height: 300px;
-    }
-    h1 {
-        width: 100%;
-        text-align: center;
-        color: rgba(23,5, 58, 0.8);
-    }
+input {
+    width: 25vw;
+    height: 3.5rem;
+    border-radius: 1rem;
+    margin-left: 1rem;
+    font-size: 2rem;
+    padding: 1rem;
+    color: var(--input-color);
 }
+
+input:focus {
+    outline: none;
+    box-shadow: 0 0 1rem var(--btn-color);
+}
+
+label {
+    font-size: 2.3rem;
+    padding: 1rem;
+}
+
+button {
+    background-color: var(--btn-color);
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+}
+
+button:hover {
+    cursor: pointer;
+}
```

Comment:
Why hard code in buttons styles instead of using button component?

Reply:
I am using a button component. Just typed in the wrong styling sheet.